src/code/game.c

```diff
@@ -55,160 +55,164 @@ void GameState_SetFBFilter(Gfx** gfx, u32 arg1) {
                 sMonoColors.envColor.g = R_FB_FILTER_ENV_COLOR(1);
                 sMonoColors.envColor.b = R_FB_FILTER_ENV_COLOR(2);
                 sMonoColors.envColor.a = R_FB_FILTER_A;
-                VisMono_Draw(&sMonoColors, &_gfx, arg1);
+                VisMono_Draw(&sMonoColors, &gfxP, arg1);
             }
         }
     }
 
-    *gfx = _gfx;
+    *gfx = gfxP;
 }
 
-void Game_Nop80173534(GameState *ctxt) {
+void Game_Nop80173534(GameState* this) {
     ;
 }
 
-void GameState_Draw(GameState *ctxt, GraphicsContext *gCtxt) {
+void GameState_Draw(GameState* this, GraphicsContext *gfxCtx) {
     Gfx* nextDisplayList;
-    Gfx* _polyOpa;
-    // Unused vars impact regalloc
-    Gfx* temp_t2;
-    u32 temp_v1;
+    Gfx* polyOpa;
+    
+    OPEN_DISPS(gfxCtx);
 
-    _polyOpa = gCtxt->polyOpa.p;
-    nextDisplayList = Graph_GfxPlusOne(gCtxt->polyOpa.p);
-    gSPDisplayList(gCtxt->overlay.p++, nextDisplayList);
+    nextDisplayList = Graph_GfxPlusOne(polyOpa = POLY_OPA_DISP);
+    gSPDisplayList(OVERLAY_DISP++, nextDisplayList);
 
     if (R_FB_FILTER_TYPE && R_FB_FILTER_ENV_COLOR(3) == 0) {
-        GameState_SetFBFilter(&nextDisplayList, (u32) gCtxt->zbuffer);
+        GameState_SetFBFilter(&nextDisplayList, (u32) gfxCtx->zbuffer);
     }
 
     if (R_ENABLE_ARENA_DBG < 0) {
         R_ENABLE_ARENA_DBG = 0;
     }
 
     gSPEndDisplayList(nextDisplayList++);
-    Graph_BranchDlist(_polyOpa, nextDisplayList);
-    gCtxt->polyOpa.p = nextDisplayList;
+    Graph_BranchDlist(polyOpa, nextDisplayList);
+    POLY_OPA_DISP = nextDisplayList;
 
-lblUnk:; // Label prevents reordering, if(1) around the above block don't seem to help unlike in OoT
-    func_800E9F78(gCtxt);
+label:
```

Comment:
revert label name and comment, at least keep the reordering part

Reply:
That was an accident sorry